packages/terminal/package.json

```diff
@@ -8,7 +8,7 @@
     "@theia/filesystem": "^0.6.0",
     "@theia/process": "^0.6.0",
     "@theia/workspace": "^0.6.0",
-    "xterm": "3.9.2"
+    "xterm": "3.13.0"
```

Comment:
@marcdumais-work Is it fine?

Reply:
Hi @akosyakov , let me run the ["ECD Theia Intellectual Property Clearance, Approach"](https://github.com/theia-ide/theia/wiki/Registering-CQs#case-new-or-updated-npm-production-dependency) on this PR branch and get back to you. ETA ~30 mins.

Reply:
Done - looks good - you can merge from a license PoV. 

There was one new extra "NOASSERTION" dependency (nugget 2.0.1), but it looks to be BSD-license after investigation:

![image](https://user-images.githubusercontent.com/25749063/57770537-88b6a800-76de-11e9-91ec-1ec667c5d5ee.png)